tests/integration/test_vulnerability_detector/test_providers/data/test_cases/cases_missing_os.yaml

```diff
@@ -53,3 +53,11 @@
   metadata:
     provider_name: 'Arch Linux'
     os: ['']
+
+- name: 'SUSE Linux Enterprise'
+  description: 'SUSE Linux Enterprise provider'
+  configuration_parameters:
+    PROVIDER: 'suse'
+  metadata:
+    provider_name: 'SUSE Linux Enterprise'
+    os: ['11-server', '12-server', '15-server', '11-desktop', '12-desktop', '15-desktop', '']
```

Comment:
Check if it is really necessary to put `''` in the last element of the `os` list.

Reply:
Yes, the `' '` as the last element is used to represent the missing os, it is present in al cases for this test.

Reply:
Fixed in https://github.com/wazuh/wazuh-qa/pull/2847/commits/b4554493f28f62ff823c8f348306c2b5aff7bbfa